### PR TITLE
Freelancer and CTT support

### PR DESCRIPTION
**Pull request by AnthonyFuller**
This adds support for Freelancer (Evergreen) and for people connecting with the Closed Technical Test.

To-do:
- [ ] Mission End screens
- [ ] Mastery
- [x] Think about how we're going to deal with allunlockables (EG differs from retail)

Possibly anything else that differs.

### CODE REVIEW

File: static/allunlockables.json [hunk 1]
```diff
@@ -17913,6 +17913,29 @@
         "IsDroppable": false,
         "Rarity": "common"
     },
+    {
```
Comment (RDIL) [inline static/allunlockables.json:17916]: Potentially breaking.

**Review by RDIL — APPROVED** [review]

**Comment by LennardF1989** [comment]
I would propose a squash of all changes to a single commit before merging.

**Comment by RDIL** [comment]
@LennardF1989 we’ll squash it when merging.

**Comment by AnthonyFuller** [comment]
> Most of the stuff that's added are EG only things that won't show up in non-eg anyways.

You'd think that, but this had been added, and it looks like this on non-EG clients:
image

It also looks almost as equally terrible on the normal build.

And yes, there is broken LOCR for the subtype "evergreen" on the suits, that part I was fine with.

**Comment by grappigegovert** [comment]
Ah interesting. Yeah, I didn't think it would do that, that's quite unfortunate

File: static/allunlockables.json (hunk 1 above)
Comment (AnthonyFuller) [reply static/allunlockables.json:17916]: Resolved in 1f14cf4158d3291103777356b6c0715b545a37d2

**Comment by moonysolari** [comment]
Did you plan to / already fix #12 with the changes on the Mission End screens on your local branch?

**Pull request closed without merging**